lib/puppet/provider/exec/powershell.rb

```diff
@@ -1,5 +1,5 @@
 require 'puppet/provider/exec'
-require_relative '../../../puppet_x/puppetlabs/powershell_manager'
+require 'puppet_x/puppetlabs/powershell_manager'
```

Comment:
Ok. So, basically, to make sure we don't run into the issue that require_relative isn't available on ruby 1.8.7, I should alter this pull request to use the same way as has been done in the puppetlabs-registry module?